app/src/main/java/com/veriff/demo/screens/login/LoginMVP.kt

```diff
@@ -0,0 +1,17 @@
+package com.veriff.demo.screens.login
+
+import com.veriff.demo.base.VeriffFlowMVP
+import com.veriff.demo.base.VeriffFlowModel
+
+interface LoginMVP : VeriffFlowMVP {
+    interface View : VeriffFlowMVP.View {
+        fun showEmailError(msg: String)
+        fun showPasswordError(msg: String)
+        fun showLoginError(msg: String)
+    }
+
+    abstract class Presenter(view: View, model: LoginModel)
```

Comment:
I would not creat a interface for final Presenter layer, as the implementation can already act as interface via hiding/exposing appropriate methods

Reply:
Just following the usual MVP style.